tools/cldr/cldr-to-icu/README.md

```diff
@@ -0,0 +1,140 @@
+relies on a pre-processing step, and the CLDR data must come from the separate
+"staging" repository (i.e. https://github.com/unicode-org/cldr-staging) or be
+pre-processed locally into a different directory.
+
+
+## Initial Setup
+
+This project relies on the Maven build tool for managing dependencies and uses
+Ant for configuration purposes, so both will need to be installed. On a Debian
+based system, this should be as simple as:
+
+```
+$ sudo apt-get install maven ant
+```
+
+You must also install an additional CLDR JAR file the local Maven repository at
+`$TOOLS_ROOT/cldr/lib` (see the README.txt in that directory for more
+information).
+
+```
+$ cd "$TOOLS_ROOT/cldr/lib"
+$ ./install-cldr-jars.sh "$CLDR_DIR"
+```
+
+## Generating all ICU data and source code
+
+```
+$ cd "$TOOLS_ROOT/cldr/cldr-to-icu"
+$ ant -f build-icu-data.xml
+```
+
+## Other Examples
+
+* Outputting a subset of the supplemental data into a specified directory:
+  ```
+  $ ant -f build-icu-data.xml -DoutDir=/tmp/cldr -DoutputTypes=plurals,dayPeriods -DdontGenCode=true
+  ```
+  Note: Output types can be listed with mixedCase, lower_underscore or UPPER_UNDERSCORE.
+  Pass `-DoutputTypes=help` to see the full list.
+
+
+* Outputting only a subset of locale IDs (and all the supplemental data):
+  ```
+  $ ant -f build-icu-data.xml -DoutDir=/tmp/cldr -DlocaleIdFilter='(zh|yue).*' -DdontGenCode=true
+  ```
+
+* Overriding the default CLDR version string (which normally matches the CLDR library code):
+  ```
+  $ ant -f build-icu-data.xml -DcldrVersion="36.1"
+  ```
+
+* Using alternate CLDR values (ex: use `alt="ascii"` values from the CLDR XML):
+
+  First, edit the `build-icu-data.xml` file where it mentions "ALTERNATE VALUES"
+  with the correctly annotated source path, target path, and locales list:
+  ```diff
+  @@ -384,6 +399,20 @@
+            <!-- ALTERNATE VALUES -->
+
+            <!-- The following elements configure alternate values for some special case paths.
+                 The target path will only be replaced if both it, and the source path, exist in
+                 the CLDR data (paths will not be modified if only the source path exists).
+
+                 Since the paths must represent the same semantic type of data, they must be in the
+                 same "namespace" (same element names) and must not contain value attributes. Thus
+                 they can only differ by distinguishing attributes (either added or modified).
+
+                 This feature is typically used to select alternate translations (e.g. short forms)
+                 for certain paths. -->
+             <!-- <altPath target="//path/to/value[@attr='foo']"
+                           source="//path/to/value[@attr='bar']"
+                           locales="xx,yy_ZZ"/> -->
+  +            <altPath target="//ldml/dates/calendars/calendar[@type='all']/dateTimeFormats/availableFormats/dateFormatItem[@alt='ascii']"
```

Comment:
Would be nice to add the extra options in the xml itself.
That way all you have to do is uncomment, not copy-paste from some readme file.

See for example the end of 
https://github.com/unicode-org/icu/blob/main/icu4c/source/data/xml/brkitr/root.xml

Reply:
Nitpick: would have been nice to have 2 commits, one for the .txt to .md conversion, and a separate one for the extra instructions.

Reply:
It's just an example, so we only need one scenario. And users might have their own custom scenarios that differ.

Yeah, if I expected followup commits based on feedback, I would have made individual commits from the beginning. I made the PR description to help reviewers. I got over optimistic and made just one commit.